src/compiler/transformers/ts.ts

```diff
@@ -2006,6 +2006,9 @@ namespace ts {
                 case TypeReferenceSerializationKind.VoidNullableOrNeverType:
                     return createVoidZero();
 
+                case TypeReferenceSerializationKind.BigIntLikeType:

```

Comment:
You also need to handle SyntaxKind.BigIntKeyword and BigIntLiterals in `serializeTypeNode`